projects/core/src/store-finder/facade/store-data.service.ts

```diff
@@ -1,11 +1,13 @@
 import { Injectable } from '@angular/core';
 import { PointOfService } from '../../model/point-of-service.model';
 
+const BASE = 10;
+
 @Injectable({
   providedIn: 'root',
 })
 export class StoreDataService {
-  readonly DECIMAL_BASE: 10;
+  readonly DECIMAL_BASE: Number = BASE;
```

Comment:
is it necessary?
will `readonly DECIMAL_BASE = 10;` fail?

Reply:
yes it fails

Reply:
reverted the change